app/styles.css

```diff
@@ -37,15 +42,19 @@ body {
 }
 
 .blinken-critical-entries {
-  text-align: left;
+  text-align: center;
 }
 
 .blinken-warning-entries {
   text-align: center;
 }
 
 .blinken-unknown-entries {
-  text-align: right;
+  text-align: center;
+}
+
+.blinken-acknowledged-entries {
+  text-align: center;
```

Comment:
Can remove all these `text-align: center`s and have a single one:

```css
.blinken-entry {
  text-align: center;
}
```

Reply:
Yep, I've done this now (in another commit).